### PR TITLE
LGA-2404 - Rotate diversity keys

### PR DESCRIPTION
**Pull request by said-moj**
## What does this pull request do?

Rotate diversity keys

### Before this can be merged:
Take a snapshot of the database before deploying this PR - https://dsdmoj.atlassian.net/wiki/spaces/laagetaccess/pages/4328980606/Create+database+snapshot

See this guide about creating your pgp keys https://dsdmoj.atlassian.net/wiki/spaces/laagetaccess/pages/4328783949/Creating+pgp+keys

```
# Copy current secret to another name
kubectl get secret diversity-keys -o yaml |  sed -e '/uid:.*/d' -e '/resourceVersion: .*/d' -e '/creationTimestamp: .*/d' -e 's/name: diversity-keys/name: previous-diversity-keys/g' | kubectl apply -f -

# Check the secret has been created successfully
kubectl get secret previous-diversity-keys -o yaml

# Put site into maintenance mode
kubectl create configmap maintenance-mode --from-literal=value=True --dry-run -o yaml | kubectl apply -f -
kubectl rollout restart deployment cla-backend-app
kubectl rollout restart deployment cla-backend-worker
# Verify site is in maintenance mode
# Public and frontend won't work whilst backend is in maintenance mode

```

### After merging:
```

# Remove the old secret
kubectl delete secret diversity-keys

# Recreate the secret from local files
kubectl create secret generic diversity-keys --from-file=private=./private.pgp --from-file=public=./public.pgp

# Recreate the pods so they pickup the new secrets
kubectl rollout restart deployment cla-backend-app
kubectl rollout restart deployment cla-backend-worker

# exec into a pod to re-encrypt the diversity data
kubectl exec -it <pod-id> bash
./manage.py diversity_reencrypt
# Enter passphrase for the PREVIOUS key when prompted
# Enter passphrase for the NEW key when prompted

# Take the site out of maintenance mode
kubectl create configmap maintenance-mode --from-literal=value=False --dry-run -o yaml | kubectl apply -f -
kubectl rollout restart deployment cla-backend-app
kubectl rollout restart deployment cla-backend-worker
```

### FINALLY 

Don't forget to add the new password to lastpass, store the old password in the comment field

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by psweeting1 — DISMISSED** [review]
LGTM

**Review by Pheonnexx — APPROVED** [review]
LGTM 🚀

File: cla_backend/apps/legalaid/utils/keys/diversity_dev_reencrypt_private.key [hunk 1]
```diff
@@ -0,0 +1,107 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```
[the remark this answers is not in the document]
Comment (jogilder) [reply cla_backend/apps/legalaid/utils/keys/diversity_dev_reencrypt_private.key:1]: This key is only used for testing

**Review by jogilder — APPROVED** [review]
LGTM

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**